code/Fluent.php

```diff
@@ -378,7 +378,7 @@ public static function domain_for_locale($locale)
      */
     public static function default_locale($domain = null)
     {
-        if ($domain === true) {
+        if (!Director::is_cli() && $domain === true) {
             $domain = strtolower($_SERVER['HTTP_HOST']);
```

Comment:
Eh, I should really back-port that to 3.x. It's annoying you can't do that easily.